### PR TITLE
Update tests structure from plugin_template

### PR DESCRIPTION
**Pull request by mdellweg**
[noissue]

This PR should only change ordering and formatting with the goal to minimize the diff of the `tests` directory wrt the corresponding one on the plugin_template.
Please compare with:
https://github.com/pulp/plugin_template/pull/34

### CODE REVIEW

File: pulp_file/tests/functional/api/test_download_content.py [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
-    FILE_FIXTURE_URL,
-    FILE_PUBLISHER_PATH,
-    FILE_REMOTE_PATH,
-)
 from pulp_file.tests.functional.utils import (
     gen_file_publisher,
     gen_file_remote,
     get_file_content_paths,
 )
+from pulp_file.tests.functional.constants import (
```
Comment (dralley) [inline pulp_file/tests/functional/api/test_download_content.py]: This should be reversed on the template side -- alphabetical ordering

File: pulp_file/tests/functional/api/test_publish.py [hunk 2]
```diff
@@ -16,15 +16,15 @@
     sync,
 )
 
+from pulp_file.tests.functional.utils import (
+    gen_file_publisher,
+    gen_file_remote,
+)
 from pulp_file.tests.functional.constants import (
```
Comment (dralley) [inline pulp_file/tests/functional/api/test_publish.py]: Same

File: pulp_file/tests/functional/utils.py [hunk 3]
```diff
@@ -6,9 +6,9 @@
 from pulp_smash import api, selectors, utils
 from pulp_smash.pulp3.constants import REPO_PATH
 from pulp_smash.pulp3.utils import (
-    gen_publisher,
```
Comment (dralley) [inline pulp_file/tests/functional/utils.py]: Alphabetize (although get_content is out of order in both)

Comment (mdellweg) [reply pulp_file/tests/functional/utils.py]: It is geT vs. geN

**Comment by kersommoura** [comment]
@mdellweg, thanks for your changes. Any time that there are modifications in tests, please, ping @pulp/qe . Then we will be able to review, and be aware of changes.

**Comment by mdellweg** [comment]
Can you teach the pulp bot to see whenever sonething in tests changed and ping / label accordingly?

**Comment by kersommoura** [comment]
@mdellweg, I will talk with the others, and perhaps we can explore this suggestion.

**Comment by kersommoura** [comment]
@rochacbruno, any other comment?

**Comment by daviddavis** [comment]
@mdellweg just FYI, we setup codeowners to automatically ping QE when tests are modified so you don't need to do it.

**Comment by dralley** [comment]
@mdellweg Please fix the alphabetical ordering and the merge conflict - thanks for the PR!

**Comment by mdellweg** [comment]
@dralley Sorry, i was blind for some of the imports. Rebased.

File: pulp_file/tests/functional/api/test_download_content.py [hunk 4]
```diff
@@ -15,16 +15,16 @@
     sync,
 )
 
-from pulp_file.tests.functional.constants import (
-    FILE_FIXTURE_URL,
-    FILE_PUBLISHER_PATH,
-    FILE_REMOTE_PATH,
-)
 from pulp_file.tests.functional.utils import (
```
Comment (dralley) [inline pulp_file/tests/functional/api/test_download_content.py]: The order of these imports (..utils and ..constants) still needs to be swapped around

File: pulp_file/tests/functional/api/test_publish.py [hunk 5]
```diff
@@ -16,15 +16,15 @@
...
     FILE_CONTENT_NAME,
     FILE_PUBLISHER_PATH,
     FILE_REMOTE_PATH,
 )
```
Comment (dralley) [inline pulp_file/tests/functional/api/test_publish.py]: same here

File: pulp_file/tests/functional/api/test_download_content.py (hunk 4 above)
Comment (mdellweg) [reply pulp_file/tests/functional/api/test_download_content.py]: That level you mean...

Comment (dralley) [reply pulp_file/tests/functional/api/test_download_content.py]: Sorry about the confusion!  Yes, that's what I meant

**Pull request merged**